### DISCUSSION TITLE
Installing Unofficial 1.1.0 Debian Packages

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2261] [Q&A]
I am running Debian (both Stable and Testing) in virtual machines and was surprised to see Testing still has 0.16.0.

If the unofficial 1.1.0 packages are installed per the instructions at https://github.com/severusseptimius/lxqt-1.1-debian, how are any updated packages (bug fixes, etc.) installed?

Thanks.

**Comment by stefonarch** [comment]
You mean the LXQt packages? It's the release version, hopefully packages will be updated to 1.2.0  in autumn.
 I'm using testing and those packages without issues, [one](https://github.com/severusseptimius/lxqt-1.1-debian/issues/8) was immediately solved.

**Reply by edwardp3** [reply]
I have never cloned a git repo. 

If the packages available at the above repo are installed via cloning its git repo, if/once updated packages eventually become available afterwards to fix bugs, etc., or perhaps a new version, _how_ are those packages updated? There isn't a Discussions section over at that repo.

**Reply by stefonarch** [reply]
The process of cloning  is quite easy as described [here](https://github.com/severusseptimius/lxqt-1.1-debian#for-debian).

Once you have this local copy you can update it easily by opening a terminal inside the directory (F4) and use git `git pull`. 
You'll have to issue the other commands again to install the updated packages. But there won't be many updates if any, the last ones were only about updating requested dependencies.

**Comment by edwardp3** [comment]
I've decided to wait until the 1.1.0 packages reach Debian's unstable repo. Hope it's soon, since the bug in 0.16.0 re grouped windows remains (only affects Debian for some reason, not seeing this in other distros).
